front/components/assistant/AssistantDetails.tsx

```diff
@@ -85,6 +85,10 @@ export function AssistantDetails({
     workspaceId: owner.sId,
     agentConfigurationId: assistantId,
   });
+  const { mutate: mutateAgentConfigurations } =
+    useProgressiveAgentConfigurations({
+      workspaceId: owner.sId,
+    });
```

Comment:
Looking a the internals of the hook it fetches quite a few things by default, you might want to "disable" it if you are just looking for the mutate method.

Reply:
(you also probably want mutateRegardlessOfQueryParams)

Reply:
+1 with @Fraggle we might not want to start all the underlying queries if we only need the mutate.